src/constants.json

```diff
@@ -14,7 +14,7 @@
   },
   "ensSubgraph": {
     "1": "https://subgrapher.snapshot.org/subgraph/arbitrum/5XqPmWe6gjyrJtFn9cLy237i4cWw2j9HcUJEXsP5qGtH",
-    "11155111": "https://api.studio.thegraph.com/proxy/49574/enssepolia/version/latest"
+    "11155111": "https://subgrapher.snapshot.org/subgraph/arbitrum/DmMXLtMZnGbQXASJ7p1jfzLUbBYnYUD9zNBTxpkjHYXV"
```

Comment:
Looks like the right "official" endpoint is `https://api.studio.thegraph.com/query/49574/enssepolia/version/latest`

Reply:
It is rate limited though

Reply:
Yes yes, It's a tricky situation 

Official and Ratelimited
`https://api.studio.thegraph.com/query/49574/enssepolia/version/latest`

Not Official and not Ratelimited
`https://subgrapher.snapshot.org/subgraph/arbitrum/DmMXLtMZnGbQXASJ7p1jfzLUbBYnYUD9zNBTxpkjHYXV`

Reply:
Since we were using the official one before, does it mean it was also rate limited ?

Reply:
Yes it was

Reply:
Thegraph team confirmed that both are same subgraphs ✅ 

we can go with `https://subgrapher.snapshot.org/subgraph/arbitrum/DmMXLtMZnGbQXASJ7p1jfzLUbBYnYUD9zNBTxpkjHYXV`



> The subgraph ID of https://api.studio.thegraph.com/query/49574/enssepolia/version/latest is indeed DmMXLtMZnGbQXASJ7p1jfzLUbBYnYUD9zNBTxpkjHYXV

> You can see that the deployment ID it returns when you query the Studio endpoint above with the query below is the same as on 
https://thegraph.com/explorer/subgraphs/DmMXLtMZnGbQXASJ7p1jfzLUbBYnYUD9zNBTxpkjHYXV?view=About&chain=arbitrum-one

> Query:
```
{
 _meta{deployment} 
}
```